backend/tests/v5/e2e/routes/teamspaces/projects/containers/tickets.test.js

```diff
@@ -510,14 +510,15 @@ const testUpdateTicket = () => {
 				expect(updatedTicket.properties).toHaveProperty('Created at');
 				expect(updatedTicket.properties).toHaveProperty('Owner');
 				delete updatedTicket.number;
+				const updatedDate = updatedTicket.properties['Updated at'];
 				delete updatedTicket.properties['Updated at'];
 				delete ticket.properties['Updated at'];
 				delete updatedTicket.properties['Created at'];
 				delete updatedTicket.properties.Owner;
-				expect({ ...updatedTicket }).toEqual({
-					...ticket,
-					...payloadChanges,
-				});
+				expect({ ...updatedTicket }).toEqual({ ...ticket, ...payloadChanges });
+
+				const ticketLog = await findOne(teamspace, 'tickets.logs', { date: new Date(updatedDate) });
```

Comment:
maybe we shoudl refactor this out to be a function as you have a copy and pasted code down at L551

Reply:
true, do you think I should take out the whole suceed clause ?

Reply:
if it's the same everywhere sure